src/zampy/datasets/utils.py

```diff
@@ -1,10 +1,24 @@
 """Shared utilities from datasets."""
+import itertools
 import urllib.request
 from pathlib import Path
+from typing import List
 from typing import Optional
+from typing import Tuple
 from typing import Union
+import cdsapi
+import pandas as pd
 import requests
 from tqdm import tqdm
+from zampy.datasets.dataset_protocol import SpatialBounds
+from zampy.datasets.dataset_protocol import TimeBounds
+
+
+PRODUCT_FNAME = {
+    "reanalysis-era5-single-levels": "era5",
+    "reanalysis-era5-land": "era5-land",
+}
+CDSAPI_CONFIG_PATH = Path.home() / ".cdsapirc"
```

Comment:
This works for now, but once we add the CAMS dataset, we cannot do this anymore: the [ADS](https://ads.atmosphere.copernicus.eu/#!/home) uses [the same file](https://ads.atmosphere.copernicus.eu/api-how-to).

In my opinion, it might be nicest to have a zampy-specific configuration file. In this file the directories can be specified, as well as the required keys for datasets that need them.

Reply:
Good idea and thanks for making an issue for it.